includes/admin/class-forms.php

```diff
@@ -1007,60 +1007,77 @@ public function render_media( $field_data ) {
 				return '';
 			}
 
-			$id = ( ! empty( $this->form_data['prefix_id'] ) ? $this->form_data['prefix_id'] : '' ) . '_' . $field_data['id'];
+			/**
+			 * Filters the media formats.
+			 *
```

Comment:
Please add the note about the media formats array formatting here. Users need to know the format, because empty array as input doesn't provide the details.